pkg/cmd/admin/router/router.go

```diff
@@ -250,6 +257,8 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out, errout io.
 		StatsPort:     defaultStatsPort,
 		HostNetwork:   true,
 		HostPorts:     true,
+
+		MaxConnections: defaultMaxConnections,
```

Comment:
Don't set a default on the client - the router should set that, not here.  The default value here should be ""